### DISCUSSION TITLE
AlertDialog with CommandBar has overlay issue

### DISCUSSION THREAD

**Discussion by ramoneguru** [twilio-labs/paste#1719] [General] [answered]
Looks like when you use the `<CommandBarContainer />` with the `<AlertDialog />` there are some overlay issues that happen

<img width="1303" alt="Screen Shot 2021-07-30 at 11 40 10 AM" src="https://user-images.githubusercontent.com/476482/127698962-e5440455-cdae-435e-adaf-d5fd6e719489.png">

**Comment by ramoneguru** [comment]
Dang, there isn't a replacement for the command bar in Paste yet it looks like... Actually, so the `<CommandBarContainer />` has the z-index set to 999 while the overlay is set to 80. I guess I could set the z-index on the command bar to 79 then

**Comment by SiTaggart** [comment, marked as answer]
Hi @ramoneguru 

In this instance you have two options, with the caveat that the Command Bar is deprecated.

1. Stop using the Command Bar. Until we've decided if there will be a replacement for the Command Bar, we'd actually just like folks to stop using it. You can create the same button layout, just allow them to be placed at the end of the form. (preferred)
2. As you have pointed out, hack the z-index of the old Command Bar with something less than 80. Ideally you might do that with a Paste zIndex design token, taken from the theme or design tokens package. (discouraged)

**Reply by ramoneguru** [reply]
I like deprecating the `CommandBar`, seems like we can use a `Card+Stack` to solve this one. Is there a way to have a `Card` fixed to the bottom of the page? Or will I need to have a custom styled component for it?

**Reply by vankhoawin** [reply]
I don't think we need to 1:1 replicate the current UX flow of the page @ramoneguru, let's work with design to figure out a stopgap until we fully migrate this page to Paste

**Reply by SiTaggart** [reply]
💯  🥇  it's almost as if @vnguyen94 works on the Design Systems team
